tests/integration/test_integration_authenticator_state.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+import os
+
+from aiounittest import AsyncTestCase
+
+from conjur_api import Client
+from conjur_api.errors.errors import HttpStatusError
+from conjur_api.models import SslVerificationMode, ConjurConnectionInfo, CredentialsData
+from conjur_api.providers import SimpleCredentialsProvider
+
+
+class ConjurUser:
+    def __init__(self, user_id: str, api_key: str):
+        self.id = user_id
+        self.api_key = api_key
+
+
+class TestEnableDisableAuthenticators(AsyncTestCase):
+    conjur_url = "https://conjur-https"
+    account = "dev"
+    isFirstTime = True
+
+    @classmethod
+    def setUpClass(cls):
+        asyncio.run(cls._add_test_data())
+
+    async def test_authenticator_enable_success(self):
+        c = await self._create_client(self.valid_user.id, self.valid_user.api_key)
+        response = await c.set_authenticator_state('authn-iam/test', True)
+
+        self.assertEqual(response, '')
```

Comment:
Empty response means success, right?

Reply:
Right. That's how the REST API works as well, with a 204 HTTP code. But the code is not usually returned when it's valid.